app/static/playlist.css

```diff
@@ -199,3 +199,33 @@ p:first-child {
   background: #000;
   color: #fff;
 }
+
+.indigenous {
+  position: fixed;
+  bottom: 20px;
+  right: 0;
+  padding: 0 75px;
+  display: none;
+  width: 685px;
+  box-sizing: border-box;
+  font-size: 34px;
+  text-align: right;
+}
+.indigenous_active {
+  display: block;
+}
+.indigenous img {
+  float: right;
+  margin-left: 65px;
+}
+.indigenous_bold {
+  font-family: 'PxGrotesk';
+  font-weight: bold;
+}
+
+@media (max-width: 720px) {
+  .indigenous {
+    width: 100%;
+    bottom: 175px;
```

Comment:
In portrait mode the padding/margins don't look even.

![Screen Shot 2020-06-02 at 10 50 49 am](https://user-images.githubusercontent.com/314298/83469908-07d07700-a4c0-11ea-85b0-ef1559bd9c6f.png)

Reply:
portrait is 720x1920 though!

Reply:
Good point!